### PR TITLE
Copy updates to align Keycard with Status and remove multiaccount

### PR DESCRIPTION
**Pull request by hesterbruikman**
# Summary

Copy changes to align Keycard UI with rest of Status UI:
- Replaces all instances of `multiaccount` in UI copy with `keys` and occasionally `root keys` if needed
- Replaces all instances of `6-digit passcode` with `passcode`, with exception of occasional helptext. Numpad + 6 dots in UI should sufiice.
- Replaces all instances of `mnemonic phrase` with `seed phrase`
...

#### Platforms
- Android

#### Areas that maybe impacted

- Keycard onboarding
- Keycard recovery
- Keycard unlock

##### Non-functional
copy

status: ready

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -687,8 +687,8 @@
 	"mobile-syncing-sheet-details": "Status tends to use a lot of data when syncing chats. You can choose not to sync when on mobile network",
 	"mobile-syncing-sheet-title": "Sync using Mobile data",
 	"more": "more",
-	"multiaccount-exists-title": "Multiaccount already exists",
-	"multiaccount-exists-content": "This multiaccount already exists. You just need to unlock it",
+	"multiaccount-exists-title": "Keys already exists",
```
Comment (vitvly) [inline translations/en.json]: Should it be singular - `exist`?

File: translations/en.json [hunk 2]
```diff
@@ -687,8 +687,8 @@
...
+	"multiaccount-exists-content": "These keys already exists. You just need to unlock them",
```
Comment (vitvly) [inline translations/en.json]: Same here.

File: translations/en.json [hunk 3]
```diff
@@ -713,20 +713,20 @@
...
 	"new-group-chat": "New group chat",
 	"new-network": "New network",
-	"new-pin-description": "Enter new 6-digit passcode",
+	"new-pin-description": "Enter new passcode",
 	"new-public-group-chat": "Join public chat",
 	"new-request": "New request",
 	"next": "Next",
-	"next-step-entering-mnemonic": "Next step is entering mnemonic phrase to import your multiaccount",
-	"next-step-generating-mnemonic": "Next step is generating mnemonic phrase for your card",
+	"next-step-entering-mnemonic": "Next step is entering seed phrase to import your keys",
```
Comment (vitvly) [inline translations/en.json]: `seed phrase` - >`a seed phrase`. Not sure though, sorry for nitpicking.

File: translations/en.json [hunk 4]
```diff
@@ -713,20 +713,20 @@
...
+	"next-step-generating-mnemonic": "Next step is generating a seed phrase for your card",
 	"no": "No",
 	"no-collectibles": "No collectibles available",
 	"no-contacts": "No contacts yet",
 	"no-extension": "No extension installed",
 	"no-keycard-applet-on-card": "No Keycard applet on card",
 	"no-messages": "No messages",
 	"no-messages-yet": "No messages yet",
-	"no-pairing-slots-available": "This card is already paired to 5 devices and cannot pair to this one. Please use one of the paired devices, log in with this card and free up pairing slots on the card",
+	"no-pairing-slots-available": "This card is already paired to its maximum of 5 devices. Please use one of the paired devices, unlock  your keys with this card and free up pairing slots on the card",
```
Comment (vitvly) [inline translations/en.json]: `unlock  your keys` contains double space

File: translations/en.json [hunk 5]
```diff
@@ -830,9 +830,9 @@
 	"recipient-code": "Enter recipient address",
 	"recover": "Recover",
 	"recover-key": "Access key",
-	"recover-keycard-multiaccount-not-supported": "Recovering keycard multiaccount with password is not supported",
-	"recover-multiaccount-warning": "Your wallet information will be exposed by importing this multiaccount.",
-	"recover-password-invalid": "This multiaccount already exists but passwords do not match",
+	"recover-keycard-multiaccount-not-supported": "Recovering keycard keys with password is not supported",
+	"recover-multiaccount-warning": "Your wallet information will be exposed by importing these keys.",
+	"recover-password-invalid": "These keys already exists but passwords do not match",
```
Comment (vitvly) [inline translations/en.json]: Probably singular `exist`

**Review by vitvly — APPROVED** [review]

File: translations/en.json (hunk 1 above)
Comment (hesterbruikman) [reply translations/en.json]: Thanks @siphiuel! Your totally right about all of these. Sorry to ask, but can you check if you access the branch and update these yourself? I'm on vacation and AFK for at least the evening. Wouldn't want to delay merge.

**Comment by vitvly** [comment]
@hesterbruikman done, sorry for delay

**Comment by yenda** [comment]
@hesterbruikman who needs to confirm these changes?

**Comment by hesterbruikman** [comment]
> 
> 
> @hesterbruikman who needs to confirm these changes?

@guylouis can you lend your eyes and take a look at these copy changes for Keycard?
cc @andmironov if still relevant tomorrow

**Comment by yenda** [comment]
@hesterbruikman also please have a second look I rebased on develop

**Comment by hesterbruikman** [comment]
@yenda Would appreciate some git guidance. How else can I check? Not seeing anything out of the ordinary in the file and there's no build.

**Comment by yenda** [comment]
@hesterbruikman should build now, it was because you didn't escape some `"` in the file

**Comment by hesterbruikman** [comment]
Looks good to me @yenda. I did notice a mistake made in a previous commit. How can I add that change to this PR? Do I add a commit to the same branch? (hesterbruikman-patch-4)

line 541   
Current: "intro-wizard-text3": "Own a Keycard? Store your keys on it; you’ll need it for transactions",
Expected: "intro-wizard-text3": "If you own a Keycard, store your keys there for enhanced security",

UPDATE:
Also noticed an error in line 72.  `"biometric-enable-keycard": "If you don't want to use your Keycard each time to access the app, enable {{bio-type-label}} sign in",`
In the UI this copy is shown with a missing translation error `[missing"en-US/IF you don't want to use your keycard each time to access the app, enable [missing {{bio-type-label}} value] sign in" translation]`
Can you confirm that this can't be changed in the translation file? I'll create another issue for it.

**Comment by yenda** [comment]
@hesterbruikman I fixed both

**Comment by yenda** [comment]
@churik can you fix e2e tests please?

**Review by andmironov — APPROVED** [review]

**Comment by flexsurfer** [comment]
@hesterbruikman @yenda is this still relevant?

**Comment by hesterbruikman** [comment]
Still relevant 👍

**Comment by flexsurfer** [comment]
@hesterbruikman lets resolve conflicts and merge then? or we need to test first?

**Comment by hesterbruikman** [comment]
Resolved all conflicts, would want to do a sanity check on a build as this has been open for a while. @flexsurfer what do I do to create a build? Wait?

**Comment by flexsurfer** [comment]
@hesterbruikman PR branch looks broken :(

**Comment by hesterbruikman** [comment]
Ugh, ok, as I don't know what causes that, it's probably more efficient if I introduce the changes again on a fresh branch. Will close this PR without merging than and will come back with a new PR in a week or so.

**Comment by flexsurfer** [comment]
@hesterbruikman let me fix this one

**Comment by flexsurfer** [comment]
@hesterbruikman ok en translation has changed a lot, so its not as easy to fix :(

**Comment by hesterbruikman** [comment]
Yes, it's fine. The file changed a lot, but it won't take much time to do it again. And Keycard integration is not in V1, so there is some time. Thanks for jumping in!

**Pull request closed without merging**